pom.xml

```diff
@@ -285,39 +285,39 @@
     <json-simple.version>1.1.1</json-simple.version>
     <json-smart.version>2.5.1</json-smart.version>
     <jsp.impl.version>9.0.90</jsp.impl.version>
-    <junit.version>5.10.3</junit.version>
-    <kerb-simplekdc.version>2.0.3</kerb-simplekdc.version>
+    <junit.version>5.11.0</junit.version>
```

Comment:
causing 
```
java.lang.NoSuchMethodError: 'java.lang.reflect.AccessibleObject org.junit.platform.commons.util.ReflectionUtils.makeAccessible(java.lang.reflect.AccessibleObject)'
	at org.eclipse.jetty.toolchain.test.jupiter.WorkDirExtension.lambda$injectFields$2(WorkDirExtension.java:78)
```

Due to non binary compatible change java.lang.NoSuchMethodError: 'java.lang.reflect.AccessibleObject org.junit.platform.commons.util.ReflectionUtils.makeAccessible(java.lang.reflect.AccessibleObject)'
	at org.eclipse.jetty.toolchain.test.jupiter.WorkDirExtension.lambda$injectFields$2(WorkDirExtension.java:78)

Just  need a new release of jetty-test-helper